src/widgets/HomePageWidgets/Hero/components/HeroSlider.tsx

```diff
@@ -0,0 +1,34 @@
+import { IHeroData } from '../model/types';
+
+export const HeroSlider = ({ hero }: IHeroData) => {
+  const { mainColor, accentColor, buttonColor } = hero;
+
+  return (
+    <>
+      <div className="pt-14 pl-5 laptop:pt-[175px] laptop:pl-[179px] flex flex-col relative">
+        <div
+          className={`font-roboto font-medium text-[${accentColor}] text-base laptop:text-[22px] mb-4`}
+        >
+          {hero.topic}
+        </div>
+        <div
+          className={`font-azeret font-bold text-2xl laptop:text-[32px] text-[#1C0041] mb-4 laptop:mb-6 w-full uppercase`}
+        >
+          {hero.title}
+        </div>
+        <div
+          className={`font-roboto font-medium text-[${mainColor}] text-xl laptop:text-2xl mb-4 laptop:mb-10 w-[335px] laptop:w-[526px]`}
+        >
+          {hero.description}
+        </div>
+
+        <button
+          type="button"
+          className={`bg-[#D5B5FF] text-[${buttonColor}] w-[130px] h-[40px] rounded-md text-sm laptop:w-[240px] laptop:h-[45px] laptop:font-medium laptop:text-xl`}
+        >
+          Shop Collection
```

Comment:
I think this button text should also be retrieved from BE (or from mocked props in your case). Potentially for different heroes, there might be different text. 

You can set 'Shop Collection' as default value e.g. `{hero.call_to_action ?? 'Shop Collection'}`